### PR TITLE
CSS

### PR DESCRIPTION
**Pull request by mariaalouisaa**
- I linked the styling and stuff to the home.js file
- Minor CSS changes to fix some missing positioning
- Put a span in the h3 for the 2 tone heading (The orange was way nicer than the brown but whatevs - accessibility trumps that i guess) 

Sorry for the changes with deleting spaces and adding semi-colons are cos I have a VS extension that tidies my code but I'll switch it off...

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -2,11 +2,12 @@ const express = require("express");
...
 
 const server = express();
-server.get('/', home.get)
 
 const staticHandler = express.static("public");
 server.use(staticHandler);
 
+server.get("/", home.get);
+
 server.get("/", (request, response) => {
```
Comment (Glombort) [inline server.js:11]: I think this can be gotten rid of now as it won't ever be shown as the home.get loads before this

File: routes/home.js [hunk 2]
```diff
@@ -1,30 +1,41 @@
...
-    for (const post of Object.values(blogPosts)) {
-        posts += `
-        <div class="blog-post"><h3>${post.user}</h3> <p>${post.message}</p></div>
-        `
-    };
-    const html = `
+  let posts = "";
+  for (const post of Object.values(blogPosts)) {
+    posts += `
+        <div class="blog-post"><div><h3>${post.user}</h3> <p>${post.message}</p></div><button class="delete">delete</button></div>
```
Comment (Glombort) [inline routes/home.js:12]: Thanks for adding the delete button, not sure how I forgot it lol:)

**Review by Glombort — APPROVED** [review]
Nice thanks for merging and resolving conflicts super helpful!!

**Comment by AsmahanM** [comment]
Looks good!

**Pull request merged**